utils/templates/termite.config.template

```diff
@@ -0,0 +1,32 @@
+[colors]
+# !!COL!{name}! colorscheme for Termite
+
+# 20% background transparency (requires a compositor)
+#background = rgba(63, 63, 63, 0.8)
+
+foreground = !!COL!{fg.srgb}!
+foreground_bold = !!COL!{br_white.srgb}!
+#foreground_dim = #888888
+background = !!COL!{bg.srgb}!
+cursor = !!COL!{fg.srgb}!
+
+# if unset, will reverse foreground and background
+#highlight = #839496
```

Comment:
similarly to what I wrote in other PRs, I suggest to change this to `br_black`.

Reply:
By not setting this it will simply use the reverse color. I think that's more readable in all cases so i recommend we leave it unset

![highlight](https://user-images.githubusercontent.com/48640805/62938809-57be4000-bdbf-11e9-8027-96c7c8b9f227.png)

Reply:
After reconsidering I think it may be a better choice indeed, at least for terminal emulators (as opposed to text editors, where I strongly prefer to use `br_black` for highlight).